### PR TITLE
Feature/view vertices edges

### LINKED ISSUE #3106: View sample data for tags & edges
**Describe the background of the requirement**
When customers get a new Nebula Graph and don't know the data details, customers always don't know how to start using our products(Nebula Graph、explorer、studio)

**Describe the detail of the requirement**
For relational database(mysql):
When we get a new database, we can know it with below:
1.show databases;
2.show tables;
3.desc <table_name> or show create <table_name>;
4.select * from <table_name> limit 10;

For nebula graph:
when we get a new database:
1.show spaces;
2.show tags & show edges;
3.show create {TAG <tag_name> | EDGE <edge_name>};
No match (v:<tag_name>) return v limit 10; (need to create index)

Can we supply a way to view sample data for tags & edges without index? then at least we can get a "vid" for query or any other operations.

**Describe the scenarios of the requirement**
1.First use of our products，don't know data details, want to understand the data structure quickly.
2.Don't know data details, want to test expand through explorer.(Need VID)

**Describe the values of the requirement**
General: nebula graph usability
Values for customer: Facilitate customers to better use our products(Nebula Graph,explorer,studio)，make it easier for customers to understand the data structure
Importance of the business project:it's common request

from @Jerry319609l

### PR DESCRIPTION
**Pull request by Shylock-Hg**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Which issue(s)/PR(s) this PR relates to?
Close #3106 
  
#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by yixinglu — COMMENTED** [review]
Generally LGTM. One thing to note is that the syntax of match may be modified in #3255

File: src/graph/optimizer/rule/PushFilterDownScanVerticesRule.cpp [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#include "graph/optimizer/rule/PushFilterDownScanVerticesRule.h"
+
+#include "common/expression/BinaryExpression.h"
```
Comment (nevermore3) [inline src/graph/optimizer/rule/PushFilterDownScanVerticesRule.cpp:8]: redundant header files

File: src/graph/optimizer/rule/PushVFilterDownScanVerticesRule.cpp [hunk 2]
```diff
@@ -0,0 +1,116 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License.
+ */
+
+#include "graph/optimizer/rule/PushVFilterDownScanVerticesRule.h"
+
+#include "common/expression/BinaryExpression.h"
```
Comment (nevermore3) [inline src/graph/optimizer/rule/PushVFilterDownScanVerticesRule.cpp:8]: redundant header files

File: src/graph/optimizer/rule/PushFilterDownScanVerticesRule.cpp (hunk 1 above)
Comment (Shylock-Hg) [reply src/graph/optimizer/rule/PushFilterDownScanVerticesRule.cpp]: Done.

File: src/graph/optimizer/rule/PushVFilterDownScanVerticesRule.cpp (hunk 2 above)
Comment (Shylock-Hg) [reply src/graph/optimizer/rule/PushVFilterDownScanVerticesRule.cpp]: Done.

**Review by CPWstatic — DISMISSED** [review]

File: src/graph/planner/plan/Query.h [hunk 3]
```diff
@@ -1212,6 +1330,8 @@ class AppendVertices final : public GetVertices {
 
   Expression* vFilter() const { return vFilter_; }
 
+  void setVFilter(Expression* filter) { vFilter_ = filter; }
```
Comment (nevermore3) [inline src/graph/planner/plan/Query.h:1333]: useless code

Comment (Shylock-Hg) [reply src/graph/planner/plan/Query.h]: Fixed.

**Review by nevermore3 — APPROVED** [review]

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**